### PR TITLE
responsive sign up completed

### LINKED ISSUE #374: 🪲[Bug]: responsive sign up
responsive design about sign up page 
display in all the devices properly ,like mobile,desktop,laptop...
image

### PR DESCRIPTION
**Pull request by Varsani2520**
## Description
[Please include a brief description of the changes or features added]
in full screen and mobile screen display different different UI

## Related Issue
[Cite any related issue(s) this pull request addresses. If none, simply state "None"]
responsive UI of sign up #374  and closed #374 

## Screenshots / GIFs (if applicable)
[Attach any relevant screenshots or GIFs demonstrating the changes]
in full screen:
image
in mobile screen:
image

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

**Comment by Varsani2520** [comment]
Hey @rishicds ,
review and merged it

**Comment by codewarnab** [comment]
> ## Description
> [Please include a brief description of the changes or features added] in full screen and mobile screen display different different UI
> 
> ## Related Issue
> [Cite any related issue(s) this pull request addresses. If none, simply state "None"] responsive UI of sign up #374 and closed #374
> 
> ## Screenshots / GIFs (if applicable)
> [Attach any relevant screenshots or GIFs demonstrating the changes] in full screen: image in mobile screen: image
> 
> ## Checklist:
> * [x]  I have performed a self-review of my code
> * [x]  I have added/updated relevant documentation (if needed)
> * [x]  I have tested the changes locally and they function as expected
> * [x]  I have ensured my code follows the project's coding standards
> 
> ## Additional Notes
> [Add any additional notes or context about the changes made]

image
just tested it locally it is not responsive your phone can you share the full screen with dimesions also for mobile

File: src/pages/user/UserRegistration.jsx
Comment (codewarnab) [inline src/pages/user/UserRegistration.jsx:1]: your pr does not contain the toggle functionality whiich was added in #386  please update your branch and give a pr

**Review by codewarnab — CHANGES_REQUESTED** [review]

**Comment by codewarnab** [comment]
> > ## Description
> > [Please include a brief description of the changes or features added] in full screen and mobile screen display different different UI
> > ## Related Issue
> > [Cite any related issue(s) this pull request addresses. If none, simply state "None"] responsive UI of sign up #374 and closed #374
> > ## Screenshots / GIFs (if applicable)
> > [Attach any relevant screenshots or GIFs demonstrating the changes] in full screen: image in mobile screen: image
> > ## Checklist:
> > 
> > * [x]   I have performed a self-review of my code
> > * [x]   I have added/updated relevant documentation (if needed)
> > * [x]   I have tested the changes locally and they function as expected
> > * [x]   I have ensured my code follows the project's coding standards
> > 
> > ## Additional Notes
> > [Add any additional notes or context about the changes made]
> 
> image just tested it locally it is not responsive your phone can you share the full screen with dimesions also for mobile

i think it will be better if you just remove the dog image in mobile screeen

**Comment by Varsani2520** [comment]
Ok I'll try

File: src/pages/user/UserRegistration.jsx
Comment (Varsani2520) [reply src/pages/user/UserRegistration.jsx:1]: Ok

**Review by Varsani2520 — COMMENTED** [review]
i remove dog img in mobile device

**Comment by Varsani2520** [comment]
Hey @codewarnab ,i solved the conflick now you are ready to merge and if yu want any changed then inform me ,i can do it

**Comment by codewarnab** [comment]
> Hey @codewarnab ,i solved the conflick now you are ready to merge and if yu want any changed then inform me ,i can do it

its  looks perfect but their was conflict again so try to fix this , or maybe just a create a new fork and put your changes their as their many new code added in that page

**Comment by Varsani2520** [comment]
@codewarnab ,i  solve the conflict ,now you can check

**Comment by Varsani2520** [comment]
Hey @codewarnab ,
i solved the conflicts 

please merge it and close the issue #374

**Comment by codewarnab** [comment]
are you facing the same error #440 ? if yes can u please fix this in your pr not a major issue for creating a different pr

**Comment by Varsani2520** [comment]
@codewarnab ,
if you merge it then automatically solve the #440

**Comment by Varsani2520** [comment]
merge and also come any error then tell me i also solve that , i already know about error is not come

**Pull request merged**